nvidia_triton/metadata.csv

```diff
@@ -0,0 +1,43 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+nvidia_triton.cache.insertion.duration,gauge,,microsecond,,"Total cache insertion duration, in microseconds",0,nvidia_triton,,
+nvidia_triton.cache.lookup.duration,gauge,,microsecond,,"Total cache lookup duration (hit and miss), in microseconds",0,nvidia_triton,,
+nvidia_triton.cache.num.entries,gauge,,,,Number of responses stored in response cache,0,nvidia_triton,,
+nvidia_triton.cache.num.evictions,gauge,,,,Number of cache evictions in response cache,0,nvidia_triton,,
+nvidia_triton.cache.num.hits,gauge,,,,Number of cache hits in response cache,0,nvidia_triton,,
+nvidia_triton.cache.num.lookups,gauge,,,,Number of cache lookups in response cache,0,nvidia_triton,,
+nvidia_triton.cache.num.misses,gauge,,,,Number of cache misses in response cache,0,nvidia_triton,,
+nvidia_triton.cache.util,gauge,,,,Cache utilization [0.0 - 1.0],0,nvidia_triton,,
+nvidia_triton.cpu.memory.total_bytes,gauge,,byte,,"CPU total memory (RAM), in bytes",0,nvidia_triton,,
+nvidia_triton.cpu.memory.used_bytes,gauge,,byte,,"CPU used memory (RAM), in bytes",0,nvidia_triton,,
+nvidia_triton.cpu.utilization,gauge,,,,CPU utilization rate [0.0 - 1.0],0,nvidia_triton,,
+nvidia_triton.energy.consumption.count,count,,,,GPU energy consumption in joules since the Triton Server started,0,nvidia_triton,,
+nvidia_triton.gpu.memory.total_bytes,gauge,,byte,,"GPU total memory, in bytes",0,nvidia_triton,,
+nvidia_triton.gpu.memory.used_bytes,gauge,,byte,,"GPU used memory, in bytes",0,nvidia_triton,,
+nvidia_triton.gpu.power.limit,gauge,,,,GPU power management limit in watts,0,nvidia_triton,,
+nvidia_triton.gpu.power.usage,gauge,,,,GPU power usage in watts,0,nvidia_triton,,
```

Comment:
You can specify the unit here, `watt` is valid https://github.com/DataDog/integrations-core/blob/c45614423f51f2c8337b99140359de0eef6f98f4/ddev/src/ddev/cli/validate/metadata_utils.py#L169